packages/bigtable/system-test/bigtable.js

```diff
@@ -27,55 +27,180 @@ var env = require('../../../system-test/env.js');
         return;
       }
 
-      async.each(tables, exec('delete'), function(err) {
+      operation
+        .on('error', done)
+        .on('complete', function() {
+          TABLE = INSTANCE.table(TABLE_NAME);
+
+          TABLE.create({
+            families: ['follows', 'traits']
+          }, done);
+        });
+    });
+  });
+
+  after(function(done) {
+    INSTANCE.delete(done);
+  });
+
+  describe('instances', function() {
+    it('should get a list of instances', function(done) {
+      bigtable.getInstances(function(err, instances) {
+        assert.ifError(err);
+        assert(instances.length > 0);
+        done();
+      });
+    });
+
+    it('should check if an instance exists', function(done) {
+      INSTANCE.exists(function(err, exists) {
+        assert.ifError(err);
+        assert.strictEqual(exists, true);
+        done();
+      });
+    });
+
+    it('should check if an instance does not exist', function(done) {
+      var instance = bigtable.instance('fake-instance');
+
+      instance.exists(function(err, exists) {
+        assert.ifError(err);
+        assert.strictEqual(exists, false);
+        done();
+      });
+    });
+
+    it('should get a single instance', function(done) {
+      var instance = bigtable.instance(INSTANCE_NAME);
+
+      instance.get(done);
+    });
+
+    it('should update an instance', function(done) {
+      var metadata = {
+        displayName: 'metadata-test'
+      };
+
+      INSTANCE.setMetadata(metadata, function(err) {
         if (err) {
           done(err);
           return;
         }
 
-        TABLE.create(done);
+        INSTANCE.getMetadata(function(err, metadata_) {
+          assert.strictEqual(metadata.displayName, metadata_.displayName);
+          done(err);
+        });
       });
     });
   });
 
-  after(function() {
-    TABLE.delete();
+  describe('clusters', function() {
+    var CLUSTER;
+
+    beforeEach(function() {
+      CLUSTER = INSTANCE.cluster(CLUSTER_NAME);
+    });
+
+    it('should retrieve a list of clusters', function(done) {
+      INSTANCE.getClusters(function(err, clusters) {
+        assert.ifError(err);
+        assert(clusters[0] instanceof Cluster);
+        done();
+      });
+    });
+
+    it('should check if a cluster exists', function(done) {
+      CLUSTER.exists(function(err, exists) {
+        assert.ifError(err);
+        assert.strictEqual(exists, true);
+        done();
+      });
+    });
+
+    it('should check if a cluster does not exist', function(done) {
+      var cluster = INSTANCE.cluster('fake-cluster');
+
+      cluster.exists(function(err, exists) {
+        assert.ifError(err);
+        assert.strictEqual(exists, false);
+        done();
+      });
+    });
+
+    it('should get a cluster', function(done) {
+      CLUSTER.get(done);
+    });
+
+    it('should update a cluster', function(done) {
+      CLUSTER.setMetadata({ nodes: 4 }, done);
+    });
+
+    it.skip('should delete a cluster', function(done) {
```

Comment:
@sduskis I'm seeing some strange behavior here - When I try to create a cluster via `BigtableInstanceAdmin.CreateCluster` I get a gRPC error stating that Operations are either not implement, unavailable or disabled. Whereas they seem to be working fine when updating a clusters metadata and creating an instance.